docs/modules/getting-started/pages/get-started-rest-api-with-docker.adoc

```diff
@@ -0,0 +1,182 @@
+= Getting Started with REST API Using Docker
+:description: This tutorial introduces you to enabling, running and using the REST API.
+
+:page-enterprise: true
+
+{description}
+
+== Before You Begin
+
+To complete this tutorial, you need the following:
+
+[cols="1a,1a"]
+|===
+|Prerequisites|Useful resources
+
+|Docker image for Hazelcast {enterprise-product-name} full distribution and an {enterprise-product-name} license
+|xref:getting-started:enterprise-overview.adoc[]
+
+|===
+
+== Run REST With Minimum Configuration
+
+== Step 1. Prepare The Configuration File
+
+Create a hazelcast.yaml (or xml) configuration file with the following content and place it in the ~/config directory on your local machine.
+
+[tabs]
+====
+XML::
++
+--
+[source,xml]
+----
+<hazelcast>
+    <license-key>your license key</license-key> <1>
+    <rest enabled="true">
+        <security-realm>restRealm</security-realm>
+    </rest>
+    <security enabled="true">
+        <realms>
+            <realm name="restRealm">
+                <authentication>
+                    <simple>
+                        <user username="restuser" password="restpassword">
+                            <role>admin</role>
+                        </user>
+                    </simple>
+                </authentication>
+                <access-control-service>
+                    <factory-class-name>com.hazelcast.internal.rest.access.DefaultAccessControlServiceFactory</factory-class-name>
+                </access-control-service>
+            </realm>
+        </realms>
+    </security>
+</hazelcast>
+----
+--
+
+YAML::
++
+[source,yaml]
+----
+hazelcast:
+  license-key: <your license key> <1>
+  rest:
+    enabled: true
+    security-realm: restRealm
+  security:
+    enabled: true
+    realms:
+      - name: restRealm
+        authentication:
+          simple:
+            users:
+              - username: 'restuser'
+                password: 'restpassword'
+                roles:
+                  - admin
+        access-control-service:
+          factory-class-name: com.hazelcast.internal.rest.access.DefaultAccessControlServiceFactory
+----
+====
+<1> Replace the `<your license key>` placeholder with your Hazelcast {enterprise-product-name} license key.
+
+== Step 2. Start Hazelcast {enterprise-product-name} Server
+
+To start the Hazelcast {enterprise-product-name} server, run the following command:
+
+If you have XML config file:
+[source,shell,subs="attributes+"]
+----
+docker run \
+    -p 5701:5701 \
```

Comment:
you should also map port 8443 for REST server.

Reply:
fixed at [efcd8b0](https://github.com/hazelcast/hz-docs/pull/1136/commits/efcd8b002c471657940150e14c777febc33a9f1b)